src/libraries/Common/src/System/Security/Cryptography/DSAOpenSsl.cs

```diff
@@ -25,7 +25,7 @@ internal static partial class DSAImplementation
         public sealed partial class DSAOpenSsl : DSA
         {
             private const int BitsPerByte = 8;
-            private Lazy<SafeDsaHandle> _key;
+            private Lazy<SafeDsaHandle> _key = null!;
```

Comment:
Why is this necessary?  Are there ctors that don't set this in other files?

Reply:
You're right, it isn't. Not sure I remember why I did this.

Reply:
Actually turns out that it's required for the OSX build. Reverting...

Reply:
And it can't be handled in the macOS-specific files?

Reply:
The only way I can think of is adding an extra `_key = null!;` assignment in  the offending constructors.

Reply:
Does that build not use this field at all? I can look at the code later, but is making this non-nullable actually wrong for objects built with those ctors?

Reply:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Security.Cryptography.OpenSsl/src/System/Security/Cryptography/DSAOpenSsl.cs#L43 

It's the usual problem, these additional ctors are initializing the field using a helper method.

Reply:
Note that both files are named `DSAOpenSSl.cs`.

Reply:
> It's the usual problem, these additional ctors are initializing the field using a helper method

Ok